src/test/compile-fail/issue28498-reject-passed-to-fn.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
```

Comment:
agh, thank you!  i must not have been reading the test's output carefully enough when I was running it by hand.

(this is what I get for trying to keep the pairs of test in sync via cut-and-paste...)